tslint.json

```diff
@@ -47,7 +47,7 @@
             "ban-keywords"
         ],
         "whitespace": [
-            true,
+            false,
```

Comment:
As previously mentioned, we aren't using tslint anymore, so, please, migrate to eslint

Reply:
Eslint has been added, I am currently going through and removing the lint errors through the repository.